src/TopDownProteomics/ProForma/ProFormaKey.cs

```diff
@@ -8,46 +8,46 @@ public class ProFormaKey
         /// <summary>
         /// The modification name
         /// </summary>
-        public static string Mod = "mod";
+        public static string Mod { get; } = "mod";
```

Comment:
My intent here is to quickly define the accessibility. It doesn't make sense for a downstream user to be able to change these fields (`ProFormaKey.Mod = "whatevs";`), since that'd break the assumed behavior. These fields/properties should be visible to them, though.

Reply:
Ah, nice catch! Maybe switching static to const is a better way to go?

Reply:
From `static` to `const` or to `static readonly`? Any opinions?

Reply:
This tends to be a religious argument: https://stackoverflow.com/questions/755685/static-readonly-vs-const. In my opinion, for this context, the ProForma publication effectively makes these string values constant. But I don't feel strongly, so static readonly would be fine too. There are also people on that thread making the case for properties over fields, but I think the situation is so unlikely that it is not worth worrying about and clear syntax is more important. All opinion, nothing to hold this PR up over :)